deny.toml

```diff
@@ -0,0 +1,24 @@
+# ********************************************************************************
+#  Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Comment:
Since you have added this file newly in 2024 I would suggest to change this accordingly.